### ISSUE TITLE
postgre, postgres, PostgresSQL, PostgreSQL

### ISSUE THREAD

**Issue by avaczi** [siscia/postgres-type#7]
Using all the above forms is rather confusing, especially for Postgres people.  The official name is PostgreSQL, while Postgres is a widely accepted nickname.  It would be very pleasing to see only these variants :)

**Comment by ypercube** [comment]
Also:  postgres-type   vs  postgre-types

It would be better if one name was used everywhere (either postgres-types or postgres-type)

**Comment by siscia** [comment]
Clearly your are rights.

You are talking about the variation that I use in the README or also about those used in the code? (I don't see many in the code...)

If it is only the README I would extremely appreciate a PR.

Finally, you would suggest to change also the name of the project? In what? postgres-types ? Or postgres-type ?

**Comment by avaczi** [comment]
@siscia I guess if it does/will incorporate multiple types (for example, `json` and `jsonb`), then the plural (postgres-types) is preferable.  In the end, if the focus remains, postgres-json would be even better :D

**Comment by siscia** [comment]
@avaczi I don't think this library is going to become anything different from what it's now.

@ypercube, do you also think that `postgres-json` would be a preferable name?

**Comment by ypercube** [comment]
Yes, since it's only about json and jsonb, I think postgres-json is more appropriate.

If there are plans to extend it in the future (say for xml), then postgres-types.
